src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -294,19 +294,16 @@ PaymentSandbox::balancesChanges (ReadView const& view) const
             switch(bt)
             {
                 case ltACCOUNT_ROOT:
-                    oldBalance = (*before)[sfBalance];
-                    newBalance = oldBalance.zeroed();
                     lowID = xrpAccount();
                     highID = (*before)[sfAccount];
+                    oldBalance = (*before)[sfBalance];
+                    newBalance = oldBalance.zeroed();
                     break;
                 case ltRIPPLE_STATE:
-                    {
-                        lowID = (*before)[sfLowLimit].getIssuer();
-                        highID = (*before)[sfHighLimit].getIssuer();
-                        oldBalance = (*before)[sfBalance];
-                        highID = (*before)[sfAccount];
-                    }
-
+                    lowID = (*before)[sfLowLimit].getIssuer();
+                    highID = (*before)[sfHighLimit].getIssuer();
+                    oldBalance = (*before)[sfBalance];
+                    newBalance = oldBalance.zeroed();
```

Comment:
Nice.  It's easier for me to think about when you modify the locals in the same order.  Good change IMO.